src/Adapter/AttributeGroup/QueryHandler/CheckIsColorGroupByIdHandler.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\AttributeGroup\QueryHandler;
+
+use PrestaShop\PrestaShop\Adapter\AttributeGroup\AbstractAttributeGroupHandler;
+use PrestaShop\PrestaShop\Core\Domain\AttributeGroup\Exception\AttributeGroupException;
+use PrestaShop\PrestaShop\Core\Domain\AttributeGroup\Query\CheckIsColorGroupById;
+use PrestaShop\PrestaShop\Core\Domain\AttributeGroup\QueryHandler\CheckIsColorGroupByIdHandlerInterface;
+
+/**
+ * Handles query which checks if provided attribute group is color group
+ */
+final class CheckIsColorGroupByIdHandler extends AbstractAttributeGroupHandler implements CheckIsColorGroupByIdHandlerInterface
```

Comment:
Maybe rename to `GetAttributeGroupIsColor`? @matks wdyt?

Reply:
Yes indeed
Maybe we can make it more generic and reusable: `GetAttributeGroupData`, `GetAttributeGroupInformations` 🤔 

I'll discuss this with @eternoendless I cant find any accurate answer about it in CQRS articles